sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -111,6 +111,17 @@ object RapidsPluginUtils extends Logging {
     }
   }
 
+  def detectMultiplePluginJars(): Unit = {
+    val classloader = ShimLoader.getShimClassLoader()
+    val rapidsJarURLs = classloader.getResources(PLUGIN_PROPS_FILENAME)
+        .asScala.toList
+    if (rapidsJarURLs.size > 1) {
+      val rapidsJars = rapidsJarURLs.map(_.toString.split("!").head).mkString(",")
+      throw new RuntimeException(s"Multiple rapids4spark jars found in the classpath: " +
+          s"$rapidsJars. Please make sure there is only one rapids4spark jar in the classpath.")
```

Comment:
@revans2 also requested that the content of the version info files is included in the error message. Are you going to do it  in a separate PR?

Reply:
Thanks, will do in this PR.